docs/tutorials/add-contracts-pallet.md

```diff
@@ -219,17 +230,16 @@ impl pallet_timestamp::Config for Runtime {
 
 /*** Add This Block ***/
 parameter_types! {
-   pub const TombstoneDeposit: Balance = deposit(
+   pub TombstoneDeposit: Balance = deposit(
       1,
       <pallet_contracts::Pallet<Runtime>>::contract_info_size()
    );
-   pub const DepositPerContract: Balance = TombstoneDeposit::get();
+   pub DepositPerContract: Balance = TombstoneDeposit::get();
```

Comment:
I've done these to make a quick fix in the tutorial. And I think a quick fix here in v3.0.0 tutorial is also needed.